components/MessageBoard/MessageList.tsx

```diff
@@ -435,7 +435,7 @@ class MessageListImpl extends JCComponent<Props, State> {
         data={this.state.messages}
         inverted={this.props.inputAt === "bottom"}
         onEndReached={!this.state.fetchingData ? () => this.getMoreMessages() : undefined}
-        style={{ height: "45vw" }}
+        style={{ height: "32vw" }}
```

Comment:
@mmacedom I checked out this PR and the component is still broken for me (I cannot see the post button):

![image](https://user-images.githubusercontent.com/48423418/110269539-f574f380-7f91-11eb-9ec5-218f5ab4b76c.png)

Scaling the height using `vw` units might not be the best solution.

I'd suggest:
```tsx
<FlatList
  style={{ height: Dimensions.get("window").height - 250 }}
/>
```

![image](https://user-images.githubusercontent.com/48423418/110269481-d4140780-7f91-11eb-8ea4-9146e8548f21.png)

Reply:
The component above the FlatList has `45px` height + `30px` padding = `75px`.

The text input is `114px` height + `40px` margin = `154px`.

`250px` = `75px` + `154px` + `20px` padding + `1px` extra because I like round numbers.

`Dimensions.get("window").height - 250` maximizes the FlatList's height, regardless of the screen resolution.